app/channels/change_task_channel.rb

```diff
@@ -10,8 +10,17 @@ def unfollow
 
   def change_task(data)
     data = data['data']
+    raise unless open_room data['room_id']
```

Comment:
Should raise some custom exception to warn the user?

Reply:
I will remove this raise because is not necessary